Overloaded get_songs and added tests

jakeHebert opened the pull request:
I overloaded get_songs() to allow users to return an array of songs for a specific album instead of all songs across their discography. I also updated the test file to test the overloaded function.

jakeHebert committed:
Update darklyrics.py

Overloaded get_songs() so it can return an array of songs from a specific album

jakeHebert committed:
Update test_get_lyrics.py

jakeHebert commented:
The test seems to be failing because python 3.5 couldn't install for the Travis CI build...

res0nance commented on darklyrics/darklyrics.py:
```diff
@@ -70,18 +70,38 @@ def get_lyrics(song, artist=''):
+    """Will allow a user to return an array of all songs from specific album."""
+    # This will still cuase some problems if a band has multiple albums with the same substring.
+    if album is not None:
+        url = get_artist_url(artist)
```
```
url = get_artist_url(artist)
        response = requests.get(url)
        soup = BeautifulSoup(response.content, 'html.parser')
        if 'not Found' in soup.title.string:
            raise LyricsNotFound()
```
This bit is repeated try moving it before the if

jakeHebert replied:
Good catch! Gonna fix that bit.

jakeHebert committed:
Update darklyrics.py

Cleaned up some repeated code.

res0nance commented on darklyrics/darklyrics.py:
```diff
@@ -70,18 +70,35 @@ def get_lyrics(song, artist=''):
+            stew_str = str(a.strong)
+            if stew_str.find(str(album)) != -1:
+                soup = BeautifulSoup(str(a), 'html.parser')
+        result = []
```
These bunch of lines are repeated as well

jakeHebert committed:
Update darklyrics.py

Did some more cleaning to get_songs(). Removed the parts that were unnecessarily repeated and reworded the docstring a bit.

res0nance reviewed (approved):
👍 Nice and clean
